pkg/core/ioutil/dump/dump.go

```diff
@@ -153,6 +159,8 @@ func Formatted(writer io.Writer, format string, data interface{}) error {
 			return err
 		}
 		return YAML(writer, data)
+	case EmptyFormat:
+		return nil
 	default:
```

Comment:
[Optional] I would suggest to define JSON explicitly. Default should fail with wrong format probably.